docs/asciidoctorj.adoc

```diff
@@ -1352,7 +1352,7 @@ public class UriIncludeProcessor extends IncludeProcessor { // <1>
 <3> `push_include` method inserts new content (retrieved from the url) in current position of document.
 
 [source]
-.Register an Inline macro processor
+.Register an Include processor
```

Comment:
Can you lowercase include here and down below? We are speaking about "include" in a general sense here, so it shouldn't be a proper noun.